x-pack/plugin/sql/src/test/java/org/elasticsearch/xpack/sql/session/ListCursorTests.java

```diff
@@ -7,15 +7,14 @@
 
 import org.elasticsearch.Version;
 import org.elasticsearch.common.io.stream.NamedWriteableRegistry;
-import org.elasticsearch.common.io.stream.Writeable.Reader;
-import org.elasticsearch.test.AbstractWireSerializingTestCase;
+import org.elasticsearch.test.AbstractWireTestCase;
 
 import java.io.IOException;
 import java.util.ArrayList;
 import java.util.Arrays;
 import java.util.List;
 
-public class ListCursorTests extends AbstractWireSerializingTestCase<ListCursor> {
+public class ListCursorTests extends AbstractWireTestCase<ListCursor> {
```

Comment:
is there a reason why AbstractWireSerializingTestCase is not preferred? from a bystander's point of view, I see ListCursor is Writeable

Reply:
This is the one time that we override how the copying is done. I made that method final to make it more clear what is going on. This is the one case of collateral damage.

Reply:
++